### PR TITLE
fix: remove get context by height

### PR DESCRIPTION
**Pull request by gene-zhan**
remove get context by height

### CODE REVIEW

File: x/evm/keeper/evm.go [hunk 1]
```diff
@@ -381,7 +381,7 @@ func (k *Keeper) ApplyMessageWithConfig(ctx cosmos.Context,
 
 	// if transaction is Aspect operational, short the circuit and skip the processes
 	if isAspectOpTx := asptypes.IsAspectContractAddr(msg.To); isAspectOpTx {
-		nativeContract := contract.NewAspectNativeContract(k.storeKey, k.getCtxByHeight, evm,
+		nativeContract := contract.NewAspectNativeContract(k.storeKey, evm,
 			ctx.BlockHeight, stateDB, k.logger)
```
Comment (dumbeng) [inline x/evm/keeper/evm.go:385]: Why passing a function here, you can just pass the block height directly.

**Review by dumbeng — CHANGES_REQUESTED** [review]

File: x/evm/keeper/evm.go (hunk 1 above)
Comment (gene-zhan) [reply x/evm/keeper/evm.go:385]: 1.   VerifyTx() requires this logic
2. The Context of the Aspect core layer cannot be resolved, and the Block Height cannot be obtained

**Review by dumbeng — APPROVED** [review]
Need to refactor this later, pass it for now

**Review by luketheart — APPROVED** [review]
LGTM.

**Pull request merged**